module/apmhttp/ignorer_test.go

```diff
@@ -65,11 +65,29 @@ func testDefaultServerRequestIgnorer(t *testing.T, ignoreURLs string, r *http.Re
 			assert.NoError(t, cmd.Run())
 			return
 		}
-		ignorer := apmhttp.DefaultServerRequestIgnorer()
-		assert.Equal(t, expect, ignorer(r))
+		defaultIgnorer := apmhttp.DefaultServerRequestIgnorer()
+		assert.Equal(t, expect, defaultIgnorer(r))
+
+		tracer := newTracer()
+		defer tracer.Close()
+
+		dynamicIgnorer := apmhttp.DynamicServerRequestIgnorer(tracer)
+		assert.Equal(t, expect, dynamicIgnorer(r))
 	})
 }
 
+func TestDynamicRequestIgnorer(t *testing.T) {
+	r := &http.Request{URL: &url.URL{Path: "/foo"}}
+	tracer := newTracer()
+	defer tracer.Close()
+
+	dynamicIgnorer := apmhttp.DynamicServerRequestIgnorer(tracer)
+	assert.Equal(t, false, dynamicIgnorer(r))
+
+	tracer.SetIgnoreTransactionURLs("/fo*")
+	assert.Equal(t, true, dynamicIgnorer(r))
+}
```

Comment:
Apart from the call to `tracer.SetIgnoreTransactionURLs`, is this testing anything that TestServerRequestIgnorer doesn't already? Doesn't look like it. If so let's drop this and add a test for Tracer.SetIgnoreTransactionURLs in the top level package if needed.